src/symmetricDifferenceWith.js

```diff
@@ -0,0 +1,29 @@
+var _curry3 = require('./internal/_curry3');
+var differenceWith = require('./differenceWith');
+var unionWith = require('./unionWith');
+
+
+/**
+ * Finds the set (i.e. no duplicates) of all elements contained in the first or
+ * second list, but not both.  Duplication is determined according to the value
```

Comment:
I suggest leaving that for a separate pull request. Also, let's see whether @CrossEye prefers one space or two. I switched from two spaces to one about ten years ago and haven't looked back. ;)